PantryNodeReact/README.md

```diff
@@ -0,0 +1,3 @@
+# PantryNode
+
+This is the typescript react frontend of Pantry Node
```

Comment:
Possibly add the steps to build and run?

Reply:
great idea adding commands to the read me